### PR TITLE
Issue #3543: [UX] Fix RTL password field style

### PR DESCRIPTION
**Pull request by klonos**
https://github.com/backdrop/backdrop-issues/issues/3543

### CODE REVIEW

File: core/modules/user/css/user.css [hunk 1]
```diff
@@ -45,10 +45,14 @@
...
 .password-toggle {
-  right: 0;
   position: absolute;
-  font-size: 80%;
   top: -2em;
+  right: 0; /* LTR */
+  font-size: 80%;
+}
+[dir="rtl"] .password-toggle {
+  right: unset;
```
Comment (quicksketch) [inline core/modules/user/css/user.css]: `unset` is relatively new (CSS 3) and doesn't have any IE support (even IE11). For the time being, could we just switch to `auto` (which is the default: https://www.w3schools.com/cssref/pr_pos_left.asp)?

File: core/modules/user/css/user.css [hunk 2]
```diff
@@ -68,6 +72,18 @@
...
+  float: left; /* LTR */
+}
+.password-strength-text {
+  padding-left: 5px; /* LTR */
+}
+[dir="rtl"] .password-strength-title {
+  float: right;
+}
+[dir="rtl"] .password-strength-text {
+  padding-right: 5px; /* LTR */
```
Comment (quicksketch) [inline core/modules/user/css/user.css]: The padding-left needs to be set back to 0 here.

File: core/modules/user/css/user.css (hunk 1 above)
Comment (klonos) [reply core/modules/user/css/user.css]: Agreed 👍 ...fixed 😉

File: core/modules/user/css/user.css (hunk 2 above)
Comment (klonos) [reply core/modules/user/css/user.css]: Yep, ...and that ` /* LTR */` should not be there. Also fixed.

**Pull request merged**